sql/core/src/main/scala/org/apache/spark/sql/execution/datasources/binaryfile/BinaryFileFormat.scala

```diff
@@ -143,6 +151,16 @@ object BinaryFileFormat {
   private[binaryfile] val CONTENT = "content"
   private[binaryfile] val BINARY_FILE = "binaryFile"
 
+  private[binaryfile]
+  val CONF_TEST_BINARY_FILE_MAX_LENGTH = "spark.test.data.source.binaryFile.maxLength"
+  /** An internal conf for testing max length. */
+  private[binaryfile] val TEST_BINARY_FILE_MAX_LENGTH = SQLConf
```

Comment:
we usually put all conf entries to `SQLConf`.

Reply:
Even for internal conf used for tests?

Reply:
ah if it's testing only, no.